packages/lodestar/src/chain/chain.ts

```diff
@@ -344,6 +344,10 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
         const justifiedBlockRoot = newState.currentJustifiedCheckpoint.root;
         const justifiedBlock = await this.db.block.get(justifiedBlockRoot);
         this.logger.important(`Epoch ${computeEpochOfSlot(this.config, justifiedBlock.slot)} is justified!`);
+
+        preJustifiedEpoch !== 0 ? this.metrics.previousJustifiedEpoch.inc() : null;
+        this.metrics.currentJustifiedEpoch.inc();
```

Comment:
Could you explain logic why we have double epoch inc?

Reply:
It's not a double inc. one is to the metric previous the other is the current